js/layout-javascript/simple-list-code.js

```diff
@@ -1220,7 +1220,7 @@ DynamicList.prototype.renderLoopHTML = function () {
           var selector = $lineClamp.length ? '[data-line-clamp]' : '.list-item-description';
           var linesToClamp = $lineClamp.length ? $lineClamp.data('line-clamp') : 3;
 
-          window.ellipsed.ellipsis(selector, linesToClamp);
+          window.ellipsed.ellipsis(selector, linesToClamp, {delimiter: ''});
```

Comment:
@YaroslavOvdii Why is this necessary?

```suggestion
          window.ellipsed.ellipsis(selector, linesToClamp, {delimiter: ''});
```

Reply:
@tonytlwu  this is needed to clamp long word correctly.
[Docs](https://github.com/nzambello/ellipsed#options).